ls1intum/Artemis
# `General`: Underscore all results correctly

### Checklist
#### General
#### Client
- [x] **Important**: I wrote super performant css

### Motivation and Context
resolves #7200 

### Steps for Testing

Go to any programming exercise with results and look in the results list. (See screenshot below)

Also check that quiz exercises are not underlined (since these results are not clickable)

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots

![image](https://github.com/ls1intum/Artemis/assets/25489463/bbe96c05-316e-4573-b79e-ca931faae193)
